recipe/build.sh

```diff
@@ -3,26 +3,19 @@ set -xeo pipefail
 
 rm -f subprojects/gtest.wrap
 EXTRA_MESON_ARGS=""
-if [[ ${variant} == "rpi" ]]; then
-  EXTRA_MESON_ARGS="${EXTRA_MESON_ARGS} -Dpycamera=enabled"
+EXTRA_MESON_ARGS="${EXTRA_MESON_ARGS} -Dgstreamer=enabled"
+EXTRA_MESON_ARGS="${EXTRA_MESON_ARGS} -Dtest=false"
+EXTRA_MESON_ARGS="${EXTRA_MESON_ARGS} -Dcam=enabled"
+EXTRA_MESON_ARGS="${EXTRA_MESON_ARGS} -Dqcam=enabled"
+EXTRA_MESON_ARGS="${EXTRA_MESON_ARGS} -Dpycamera=enabled"
+EXTRA_MESON_ARGS="${EXTRA_MESON_ARGS} -Dv4l2=true"
+if [[ ${variant} == "rpi_fork" ]]; then
   EXTRA_MESON_ARGS="${EXTRA_MESON_ARGS} -Dpipelines=rpi/vc4,rpi/pisp"
   EXTRA_MESON_ARGS="${EXTRA_MESON_ARGS} -Dipas=rpi/vc4,rpi/pisp"
```

Comment:
I would encourage you to build all the default pipelines and IPAs with these in addition to them. 

The the rpi variant becomes a clear superset of the functionality of the upstream variant.

Reply:
for me it's ok, thank you!